packages/apollos-church-api/package.json

```diff
@@ -49,17 +50,18 @@
     "prettier": "1.12.1",
     "randomcolor": "0.5.3",
     "apollo-server-express": "2.4.8",
-    "express": "^4.17.0"
+    "express": "^4.17.0",
+    "@apollosproject/data-connector-cloudinary": "^1.1.0-beta.3"
```

Comment:
why is this added?

Reply:
Bad merge, again. You can probably remove.